### ISSUE TITLE
LightStereo fine-tuning with max_disp > 192

### ISSUE THREAD

**Issue by valeriopaolicelli** [XiandaGuo/OpenStereo#170]
Hi,
First of all thank you for this repo!
I would like to perform a fine-tuning of your LightStereo models, but using my custom dataset which has a wider disparity range (max disp > 192px). Is it possible to start from your pretrained models? 

Thank you

**Comment by saadehmd** [comment]
Yeah!! I also had the same question. I have the baseline camera distances much greater than usual cases. Around 4 m. I have estimated I need a max_disparity of around 500 - 800 pixels. Wondering If I really need fine-tuning just to target a bigger Max_disparity, or is there some post-processing trick I could do ?
